CONTRIBUTING.md

```diff
@@ -64,8 +64,31 @@ Instructions for adding style to a component:
 We have adopted the [ducks pattern](https://www.freecodecamp.org/news/scaling-your-redux-app-with-ducks-6115955638be/) for structuring our application.
 
 ### Testing
+We are using [Jest](https://jestjs.io/) as our testing framework foundation along with [Testing Library](https://testing-library.com/) for testing UI components.
+
```

Comment:
Hi gilles - can we add more on the *how* instead of describing the *why* here? I think it would be helpful to point to a simple test we have written or maybe even walk through how to go about using our testing approach with an example. See: https://github.com/patternfly/patternfly-react/blob/master/CONTRIBUTING.md for an example of what I mean.